certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Data247",
+    "description": "Welcome to the Data247 API! You can use our API to access the various data services we provide.",
```

Comment:
Could this description please be expanded on a bit? The description should explain the main value and purpose of this connector, in clear terms to the user, so they're aware of what it is capable of.

Reply:
this is already done, so resolving the conversation.